--- src/gtk2/gui_dialogs.c.str	2009-12-04 02:16:13.000000000 +0100
+++ src/gtk2/gui_dialogs.c	2009-12-04 02:16:33.000000000 +0100
@@ -56,6 +56,7 @@
                                     GTK_DIALOG_MODAL,
                                     GTK_MESSAGE_QUESTION,
                                     GTK_BUTTONS_YES_NO,
+				    "%s",
                                     message);
     button = gtk_dialog_run(GTK_DIALOG(dialog));
     switch(button) {
@@ -95,6 +96,7 @@
                                     GTK_DIALOG_MODAL,
                                     GTK_MESSAGE_INFO,
                                     GTK_BUTTONS_CLOSE,
+				    "%s",
                                     wordwrap(message, 60));
     gtk_dialog_run(GTK_DIALOG(dialog));
     gtk_widget_destroy(dialog);
